### DISCUSSION TITLE
SPA with Lucky how to/tutorial?

### DISCUSSION THREAD

**Discussion by petr-fischer** [luckyframework/lucky#1813] [General] [answered]
Is there any doc how to write Single Page Application with actions that reload only a part (fragment) of the page - in Lucky? Thanks!

**Comment by jwoertink** [comment, marked as answer]
There's no current guides, but I wrote this a while back https://dev.to/jwoertink/quick-start-with-react-on-lucky-1mn8 and then wrote this recently https://dev.to/jwoertink/getting-lucky-with-htmx-57b3

You might be able to get enough info from those, but if not, the Discord server has people hanging out.
